utilities/write_batch_with_index/write_batch_with_index_internal.h

```diff
@@ -314,44 +317,10 @@ class WriteBatchWithIndexInternal {
   //   and return kMergeInProgress
   // If batch does not contain this key, return kNotFound
   // Else, return kError on error with error Status stored in *s.
-  WBWIIteratorImpl::Result GetFromBatch(WriteBatchWithIndex* batch,
-                                        const Slice& key, std::string* value,
-                                        Status* s) {
-    return GetFromBatch(batch, key, &merge_context_, value, s);
-  }
-  WBWIIteratorImpl::Result GetFromBatch(WriteBatchWithIndex* batch,
-                                        const Slice& key,
-                                        MergeContext* merge_context,
-                                        std::string* value, Status* s);
-
-  // Merge with no base value
-  Status MergeKey(const Slice& key, const MergeContext& context,
-                  std::string* result) const;
-  Status MergeKey(const Slice& key, std::string* result) const {
-    return MergeKey(key, merge_context_, result);
-  }
-
-  // Merge with plain base value
-  Status MergeKey(const Slice& key, const Slice& value,
-                  const MergeContext& context, std::string* result) const;
-  Status MergeKey(const Slice& key, const Slice& value,
-                  std::string* result) const {
-    return MergeKey(key, value, merge_context_, result);
-  }
-
-  size_t GetNumOperands() const { return merge_context_.GetNumOperands(); }
-  MergeContext* GetMergeContext() { return &merge_context_; }
-  Slice GetOperand(int index) const { return merge_context_.GetOperand(index); }
-
- private:
-  const ImmutableOptions& GetCFOptions() const;
-  std::tuple<Logger*, Statistics*, SystemClock*> GetStatsLoggerAndClock(
-      const ImmutableOptions& cf_opts) const;
-
-  DB* db_;
-  const DBOptions* db_options_;
-  ColumnFamilyHandle* column_family_;
-  MergeContext merge_context_;
+  static WBWIIteratorImpl::Result GetFromBatch(
```

Comment:
Just a minor thought. While I know that this is already in an _internal class, I'm just wondering if we want to name this `GetFromBatchInternal` or `GetFromBatchImpl` to make it clearer that this isn't a public API. I think It would make internal code search a bit easier. :)

Reply:
I think I'd rather keep this in sync with `WriteBatch` / `WriteBatchInternal`, where we use the same method name in the two classes. Also, now that the methods are static helpers, clients will call them with the class name prepended (as `WriteBatchWithIndexInternal::GetFromBatch` etc.), so there's no ambiguity